src/internal/styles/forms/constants.scss

```diff
@@ -6,7 +6,7 @@
 @use '../utils/theming' as theming;
 @use '../tokens' as awsui;
 
-$control-border-width: awsui.$border-width-field;
+$control-border-width: awsui.$border-width-button;
```

Comment:
Should we remove this intermediate constant and use `awsui.$border-width-button` in all remaining places directly?

Reply:
I love the idea. Now it's not as useful as before. Will update